src/plugins/mdraid.c

```diff
@@ -275,7 +275,7 @@ static GHashTable* parse_mdadm_vars (const gchar *str, const gchar *item_sep, co
     return table;
 }
 
-static BDMDExamineData* get_examine_data_from_table (GHashTable *table, gboolean free_table, GError **error) {
+static BDMDExamineData* get_examine_data_from_table (GHashTable *table, gboolean free_table, G_GNUC_UNUSED GError **error) {
```

Comment:
This should be squashed with the previous commit. And I believe the `error` should actually be used.

Reply:
Squashed. We could perhaps make use of `error` by defining couple of keys as mandatory. Just a thought.

Reply:
We don't have to do that now.